app/scripts/controllers/network/network-controller.ts

```diff
@@ -1067,7 +1070,7 @@ export class NetworkController extends EventEmitter {
       referrer,
       source,
     }: {
-      setActive: boolean;
+      setActive?: boolean;
```

Comment:
Should this be added to the TypeScript migration PR?

Reply:
Yes, good call.